spring-cloud-dataflow-server/pom.xml

```diff
@@ -19,6 +19,11 @@
 	</properties>
 
 	<dependencies>
+		<dependency>
+			<groupId>io.projectreactor</groupId>
+			<artifactId>reactor-core</artifactId>
+			<version>3.3.0.RELEASE</version>
```

Comment:
Similar to the previous comment on version management.

Reply:
Actually this dep doesn't belong here. It needs to come from a transitive deps, which currently is only a cf java client.